### PR TITLE
feat: add information traces

### PR DESCRIPTION
**Pull request by mvicrob**

### CODE REVIEW

**Review by Coleim — DISMISSED** [review]

File: models/BudgetResults.v1.json [hunk 1]
```diff
@@ -8,6 +8,12 @@
     },
     "funding_plan": {
       "$ref": "./FundingResults.v1.json"
+    },
+    "informations": {
```
Comment (ahom) [inline models/BudgetResults.v1.json:12]: I don't think there is an "s" at the end of information.

Comment (ahom) [reply models/BudgetResults.v1.json:12]: I have the feeling that "information" is weird here, here are a few proposition that I have in mind:
- logs
- traces
- notifications
- messages

File: models/InformationLevel.v1.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "type": "string",
+  "title": "InformationLevel",
+  "enum": [
+    "INFO",
+    "WARN",
+    "ERROR"
```
Comment (ahom) [inline models/InformationLevel.v1.json:7]: Can we really have error messages here?

File: models/BudgetResults.v1.json (hunk 1 above)
Comment (mvicrob) [reply models/BudgetResults.v1.json]: logs is good yes

**Review by ahom — APPROVED** [review]

**Pull request merged**